guidance/metadata-template.json

```diff
@@ -0,0 +1,121 @@
+[
+  {
+    "columnTitle": "file_path",
```

Comment:
Should we name it as property, as this isn't a tdr file header or column title?
  
`"property": "file_path",`